integration-tests/test.sh

```diff
@@ -45,6 +45,8 @@ echo "$out"
 
 if [[ $os == "arch" ]]; then
     egrep -q 'Count: 84, Failed: 0, Skipped: 3' <<<"$out"
+elif [[ $os == "alpine3" ]]; then
```

Comment:
Can the tests be re-written in a way that avoids this  if condition?

Want to avoid maintaining too many separate test counts.

Reply:
This PR adds an OpenRC (Alpine) specific attribute so I added tests only to the Alpine goss file.

I think it might be better to refactor the integration tests to move the Dockerfiles under the OS directories and split `test.sh` to have an OS specific part in the OS directory that will include the expected counts of pass/failed/skipped tests.

Reply:
@aelsabbahy I hope based on my comment above that the alpine3 `if` condition is OK.

If you still object, what do you think of my idea to refactor integration tests to take expected results from file in OS directory? If you approve I could open a separate PR for this work.